### PR TITLE
Create item-selector.html

### PR DESCRIPTION
**Pull request by hzik**
Originally it's for selecting published items from different projects. Please see the README file.

### CODE REVIEW

**Comment by hzik** [comment]
sure, done :)

File: Item-selector/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Configuration
+
+```json
+{
+    "projectid": "302946ce-a441-00e5-3dba-ec6ccc479168",
+    "filter": "system.type=article"
+}
+```
+
+You need to specify the `projectid` parameter in order to make the element work. The optional `filter` parameter is for filtering just subitems of your project and you can use any filtering described in our documentation (separated by &):
```
Comment (Simply007) [inline Item-selector/README.md]:
```suggestion
You need to specify the `projectid` parameter in order to make the element work. The optional `filter` parameter is for filtering just subitems of your project and you can use any filtering described in our [documentation](https://developer.kenticocloud.com/reference#content-filtering) except [`depth`](https://developer.kenticocloud.com/reference#section-linked-content-depth) and [`elements`](https://developer.kenticocloud.com/reference#projection) (separated by &):
```

File: Item-selector/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Output
+
+The JSON output of such an element would look like this:
+
+```json
+"item_selector": {
+        "type": "custom",
+        "name": "item selector",
+        "value": "[
+                   {\"codename\":\"article_1\",\"name\":\"Article 1\",\"type\":\"article\"},
```
Comment (Simply007) [inline Item-selector/README.md]: I would consider adding language codename and/or guid in the selected response.

File: Item-selector/README.md (hunk 1 above)
Comment (Simply007) [reply Item-selector/README.md]: I would also note what language is retrieved from the project and I would link the documentation describing the language fallbacks:
* https://docs.kenticocloud.com/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks

File: Item-selector/README.md (hunk 2 above)
Comment (hzik) [reply Item-selector/README.md]: done!

File: Item-selector/README.md (hunk 1 above)
Comment (hzik) [reply Item-selector/README.md]: done, thanks!

File: gallery-src/content/elements/ItemSelector.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Item selector",
+  "description": "This element offers items from another project to be linked with your item",
+  "thumbnailUrl": "https://amend.cz/item-selector.png",
+  "readmeUrl": "https://github.com/Kentico/custom-element-samples/tree/master/Item-selector"
```
Comment (Simply007) [inline gallery-src/content/elements/ItemSelector.json]:
```suggestion
  "readmeUrl": "https://github.com/Kentico/kontent-custom-element-samples/tree/master/Item-selector"
```

File: Item-selector/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+```json
+{
+    "projectid": "302946ce-a441-00e5-3dba-ec6ccc479168",
+    "filter": "system.type=article"
+}
+```
+
+You need to specify the `projectid` parameter in order to make the element work. The optional `filter` parameter is for filtering just subitems of your project and you can use any filtering described in our documentation (separated by &):
+
+https://developer.kenticocloud.com/reference#content-filtering
```
Comment (Simply007) [inline Item-selector/README.md]:
```suggestion
[Content filtering documentation](https://developer.kenticocloud.com/reference#content-filtering)
```

File: Item-selector/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Within the filter you can also specify what language of your items you want to retrieve (a default language is returned ootb) - https://docs.kenticocloud.com/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks
```
Comment (Simply007) [inline Item-selector/README.md]:
```suggestion
Within the filter you can also specify what language of your items you want to retrieve (a default language is returned OOTB) - [https://docs.kontent.ai/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks](https://docs.kontent.ai/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks)
```

File: Item-selector/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+You can test this element by configuring https://kentico.github.io/custom-element-samples/Item-selector/item-selector.html url for your custom element.
```
Comment (Simply007) [inline Item-selector/README.md]:
```suggestion
You can test this element by configuring https://kentico.github.io/kontent-custom-element-samples/Item-selector/item-selector.html url for your custom element.
```

**Pull request merged**